src/main/shortcutHandler.js

```diff
@@ -0,0 +1,295 @@
+      ['paragraphHeading2', 'CmdOrCtrl+2'],
+      ['paragraphHeading3', 'CmdOrCtrl+3'],
+      ['paragraphHeading4', 'CmdOrCtrl+4'],
+      ['paragraphHeading5', 'CmdOrCtrl+5'],
+      ['paragraphHeading6', 'CmdOrCtrl+6'],
+      ['paragraphUpgradeHeading', 'CmdOrCtrl+='],
+      ['paragraphDegradeHeading', 'CmdOrCtrl+-'],
+      ['paragraphTable', 'CmdOrCtrl+T'],
+      ['paragraphCodeFences', 'CmdOrCtrl+Alt+C'],
+      ['paragraphQuoteBlock', 'CmdOrCtrl+Alt+Q'],
+      ['paragraphMathBlock', 'CmdOrCtrl+Alt+M'],
+      ['paragraphHtmlBlock', 'CmdOrCtrl+Alt+H'],
+      ['paragraphOrderList', 'CmdOrCtrl+Alt+O'],
+      ['paragraphBulletList', 'CmdOrCtrl+Alt+U'],
+      ['paragraphTaskList', 'CmdOrCtrl+Alt+X'],
+      ['paragraphLooseListItem', 'CmdOrCtrl+Alt+L'],
+      ['paragraphParagraph', 'CmdOrCtrl+0'],
+      ['paragraphHorizontalLine', 'CmdOrCtrl+Alt+-'],
+      ['paragraphYAMLFrontMatter', 'CmdOrCtrl+Alt+Y'],
+
+      // format menu
+      ['formatStrong', 'CmdOrCtrl+B'],
+      ['formatEmphasis', 'CmdOrCtrl+I'],
+      ['formatInlineCode', 'CmdOrCtrl+`'],
+      ['formatStrike', 'CmdOrCtrl+D'],
+      ['formatHyperlink', 'CmdOrCtrl+L'],
+      ['formatImage', 'CmdOrCtrl+Shift+I'],
+      ['formatClearFormat', 'Shift+CmdOrCtrl+R'],
+
+      // window menu
+      ['windowMinimize', 'CmdOrCtrl+M'],
+      ['windowCloseWindow', 'CmdOrCtrl+Shift+W'],
+
+      // view menu
+      ['viewToggleFullScreen', isOsx ? 'Ctrl+Command+F' : 'F11'],
+      ['viewChangeFont', 'CmdOrCtrl+.'],
+      ['viewSourceCodeMode', 'CmdOrCtrl+Alt+S'],
+      ['viewTypewriterMode', 'CmdOrCtrl+Alt+T'],
+      ['viewFocusMode', 'CmdOrCtrl+Shift+F'],
+      ['viewToggleSideBar', 'CmdOrCtrl+J'],
+      ['viewToggleTabBar', 'CmdOrCtrl+Alt+B'],
+      ['viewDevToggleDeveloperTools', isOsx ? 'Alt+Command+I' : 'Ctrl+Shift+I'],
+      ['viewDevReload', 'CmdOrCtrl+R']
+    ])
+
+    // fix non-US keyboards
+    this.mnemonics = new Map()
+    this.fixLayout()
+
+    // load user-defined keybindings
+    this.loadLocalKeybindings()
+  }
+
+  fixLayout () {
+    const lang = getKeyboardLanguage()
+    switch (lang) {
+      // fix aidou and inline code
+      case 'ch':
+      case 'de':
+      case 'dk':
+      case 'fi':
+      case 'no':
+      case 'se':
+        this.fixInlineCode()
+
+        if (!isOsx) {
+          this.fixAidou()
+        }
+        break
+
+      // fix aidou only
+      case 'es':
+      case 'hr':
+      case 'it':
+      case 'pl':
+      case 'pt':
+        if (!isOsx) {
+          this.fixAidou()
+        }
+        break
+
+      // custom layouts
+      case 'bg':
+        this.mnemonics.set('CmdOrCtrl+/', 'CmdOrCtrl+8')
+        this.fixInlineCode()
+        break
+    }
+  }
+
+  fixAidou () {
+    this.mnemonics.set('CmdOrCtrl+/', 'CmdOrCtrl+7')
+  }
+
+  // fix dead backquote key on layouts like German
+  fixInlineCode () {
+    this.keys.set('formatInlineCode', 'CmdOrCtrl+Shift+B')
+  }
+
+  loadLocalKeybindings () {
```

Comment:
Do we need to add some docs about how to add custom keybindings in the docs folder, and refer in the `README.md` file.

Reply:
I'll will do that and create a `misc` folder so we can create templates for the end-user documentation and merge these files later.